.github/workflows/cd.yml

```diff
@@ -56,6 +75,18 @@ jobs:
     runs-on: self-hosted
     needs: build
     steps:
+      - name: On staging env
+        env:
+          REMOTE_URL: developer-example.staging.cosmian.com
+          DOCKER_COMPOSE_FILE: saas_applications_examples/docker-compose.yml
```

Comment:
we'll have two different docker-compose files for prod and staging, mapping port differently for example. 
So we will create on the VM the same architecture as on other demos : 
saas_applications_examples/
      prod/ -> docker-compose.yml
      staging/ -> docker-compose.yml
      
      The path should then be : saas_applications_examples/staging/docker-compose.yml